persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -49,6 +51,21 @@ UPDATE Message
 SET visibility = ?, text_body = ?
 WHERE id = ? AND conversation_id = ?;
 
+updateNotificationFlagByConversation:
+UPDATE Message
+SET shouldNotify = 0
+WHERE conversation_id = ? AND shouldNotify = 1;
+
+updateNotificationFlagForAll:
```

Comment:
`updateNotificationFlagForAll` and `updateNotificationFlagByConversation` are not used, cause it may be "too big" operations.
remove it